workers/unity/Packages/com.improbable.gdk.tools/.DownloadCoreSdk/Program.cs

```diff
@@ -22,14 +22,21 @@ private static void Main(string[] args)
             var schemaStdLibDir = Path.GetFullPath(args[2]);
             var nativeDependenciesPath = Path.GetFullPath("./Assets/Plugins/Improbable/Core");
             var managedDependenciesPath = Path.GetFullPath("./Assets/Plugins/Improbable/Sdk");
+            var iOSManagedDependenciesPath = Path.GetFullPath("./Assets/Plugins/iOS");
```

Comment:
Unfortunately, if we move this to `Improbable`, Unity will not pick it up automatically as iOS-only. We can do that, but it will result in errors when first opening Editor, until iOS library is unchecked for other platforms

Reply:
Even if its `Improbable/iOS` ? 😭

Reply:
Yup :( It only registers it at the Asset/Plugin/iOS

Reply:
yeah, unity is a bit weird with that. we should move it to that folder to make sure it gets recognized correctly

Reply:
This will be resolved back to original structure with https://github.com/spatialos/gdk-for-unity/pull/517

Reply:
You'll need to update the `.gitignore` to ignore the plugins in `iOS` as well

Reply:
@jamiebrynes7 this shouldn't be an issue when we merge #517 in here first